### ISSUE TITLE
Train on my own language

### ISSUE THREAD

**Issue by vndee** [freesunshine0316/neural-graph-to-seq-mp#18]
Hi @freesunshine0316,
Thanks for your great work. I've used your code to train on my custom language (Vietnamese). I have more than 3000 samples in dataset. Using your pretrained silver model to train on my dataset continously. But when I load pretrained model I realized that I can't modify the input word embedding layer with a specific size of more than 27k words. So I remove some word vector in vectorRNN.txt and add word vector for Vietnamese, which was collected from Fasttext. During training, I see accuracy is around 0.92. But when I ran the decode script, I've got an unexpected output for all samples in the training set. It can predict text length, number token exactly, and the other is always the same <UNK> token.
Can you give me a guide or advice for training in my own language?. Thanks.

**Comment by freesunshine0316** [comment]
@vndee Hi, thank you for your interest and sorry for the late reply.

The pretrained model and vecvtorRNN.txt.st are optimized together, and thus a naive hacking technique you mentioned won't work. If you have to use a new embedding file, the only way is to train the model from scratch.

**Comment by freesunshine0316** [comment]
Hi @vndee 

I wrote my previous comment during a road trip, and I just rephrase it for better understandable. 
In general, you have to train a model from scratch for your case. Hope your problem has been solved.
